README.md

```diff
@@ -1,2 +1,157 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Microverse-Trial-Project-Code-Review <a name="about-project"></a>
+
+This project is about reviewing my Microverse Trials project. I reviewed it by applying the skills i learnt during my first week in Microverse bootcamp.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://css.com/">CSS</a></li>
+  </ul>
+</details>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A code editor or an IDE installed on your computer (e.g., Visual Studio Code, Sublime Text, Atom).
+2. Git installed on your system to clone the repository.
+3. A web browser to view and test the project.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git@github.com:Ayokunnumi1/Microverse-Trial-Project-Code-Review.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="author"></a>
+<a name="authors">Ayokunnumi Omololu</a>
+* GitHub: [@Ayokunnumi1][https://github.com/Ayokunnumi1]
+* Twitter: [@Ayokunnumi1] [https://twitter.com/AyokunnumiA]
+* LinkedIn: [Ayokunnumi Omololu] [https://www.linkedin.com/in/ayokunnumiomololu]
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Footer**
+- [ ] **Interactive Feedbacks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly follow me on github.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to give thanks to Microverse for creating this project to help improve my skills.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
kindly update your read me file. Add how to install your project to the section.